yoyodyne/evaluators.py

```diff
@@ -203,10 +200,27 @@ def _compute_ser(
         preds: List[str],
         target: List[str],
     ) -> float:
-        errors = _edit_distance(preds, target)
+        errors = self._edit_distance(preds, target)
         total = len(target)
         return errors / total
 
+    def _edit_distance(self, x: List[str], y: List[str]) -> int:
+        idim = len(x) + 1
+        jdim = len(y) + 1
+        table = numpy.zeros((idim, jdim), dtype=numpy.uint16)
+        table[:, 0] = range(idim)
+        table[0, :] = range(jdim)
+        for i in range(1, idim):
+            for j in range(1, jdim):
+                if x[i - 1] == y[j - 1]:
+                    table[i][j] = table[i - 1][j - 1]
+                else:
+                    c1 = table[i - 1][j]
+                    c2 = table[i][j - 1]
+                    c3 = table[i - 1][j - 1]
+                    table[i][j] = min(c1, c2, c3) + 1
+        return table[-1][-1]
```

Comment:
Suggest you cast this to `int` before returning because we don't need the `uint16` precision (or any other numpy weirdness). I don't know if it's actually important...

Reply:
Done. (Also updated README)